operator/docs/examples/custom-authenticator.yaml

```diff
@@ -0,0 +1,31 @@
+# To create this cluster, run:
+# * export GUEST_CLUSTER_NAME="foobar"
+# * envsubst < auth-stage2.yaml | kubectl --kubeconfig $KUBECONFIG apply -f -
+
+apiVersion: kit.k8s.sh/v1alpha1
```

Comment:
We have tested this example spec right?

Reply:
Yes. Forgot to rename `auth-stage2.yaml` back to `custom-authenticator.yaml` after though. I'll update it.